keycloak/keycloak-ui
# Updated message for "Remove imported users?" modal - Issue 2583

## Motivation
Closes https://github.com/keycloak/keycloak-ui/issues/2583

## Verification Steps
1. Go to `User federation`.
2. Create a new item and navigate to it.
3. Select `Remove imported` from the `Action` dropdown.
4. Verify that text in the modal has been updated.

## Checklist:
